lib/js/sportsday.js

```diff
@@ -211,7 +211,6 @@
         // failure
         apiErr(res);
       });
-    setCookie("stopSurveyAlert", false, 0.5); // cookie to store if user wants survey -- 0.5 of a day = 12 hours
```

Comment:
This line meant that every time the page loaded the `stopSurveyAlert` cookie would be reset to false, meaning on every new page load the survey modal would be shown after 10 visits, rendering the cookie useless (unless the user never refreshed the site entirely). Removing this line means that the cookie doesn't exist by default, but is added when the modal is dismissed